ci_tools/check_slots.py

```diff
@@ -26,82 +28,132 @@
 missing_attribute_nodes = []
 missing_from_all = []
 
+ErrorCollection = {
```

Comment:
Generally in Python CamelCase is used for type names. This is misleading here. Would you mind renaming this dictionary. E.g. `error_collection`